### DISCUSSION TITLE
How do you use the `updated_at` mapper?

### DISCUSSION THREAD

**Discussion by Richard-Degenne** [keycloak/keycloak#19105] [Q&A]
Hello,

I have a use-case in which I want my application to synchronize its own database with the user's userinfo whenever the user gets updated.

I have noticed that a `updated_at` mapper exists and points to a `updatedAt` attribute, but it seems to me that Keycloak does not manage that attribute.

What is the best way to get that attribute to work? Do I have to write a custom SPI to do it? It seems to me that this should be a default feature, rather than some custom plugin thing, and I feel like I'm missing something.

**Comment by pautx24** [comment]
Hey! I've got a similar use case. Have you found an ellegant solution rather than update that attribute manually?

**Comment by Richard-Degenne** [comment]
Hey @pautx24! Unfortunately, I have not found an answer, and I had to make do without this for the time being. But if someone else comes around with an answer, I'm still here for it!
